.gitignore

```diff
@@ -5,6 +5,10 @@
 /vendor/
 ###< symfony/framework-bundle ###
 
+###> PhpStorm ###
+/.idea/
+###< PhpStorm ###
+
```

Comment:
Please remove this. You should use a gitignore global for that https://help.github.com/articles/ignoring-files/